test/functional/Java22andUp/src/org/openj9/test/jep454/upcall/InvalidUpCallTests.java

```diff
@@ -302,16 +302,16 @@ public void test_InvalidLinkerOptions_isTrivial_1() throws Throwable {
 	}
 
 	@Test(expectedExceptions = IllegalThreadStateException.class, expectedExceptionsMessageRegExp = ".* wrong thread state for upcall")
-	public void test_InvalidLinkerOptions_isTrivial_2() throws Throwable {
+	public void test_InvalidLinkerOptions_isCritical_2() throws Throwable {
 		FunctionDescriptor fd = FunctionDescriptor.of(JAVA_INT, JAVA_INT, ADDRESS);
 		MemorySegment functionSymbol = nativeLibLookup.find("captureTrivialOptionByUpcallMH").get();
 		MethodHandle mh = linker.downcallHandle(functionSymbol, fd, Linker.Option.critical(false));
 
 		try (Arena arena = Arena.ofConfined()) {
-			MemorySegment upcallFuncAddr = linker.upcallStub(UpcallMethodHandles.MH_captureTrivialOption,
+			MemorySegment upcallFuncAddr = linker.upcallStub(UpcallMethodHandles.MH_captureCriticalOption,
 					FunctionDescriptor.of(JAVA_INT, JAVA_INT), arena);
 			int result = (int)mh.invoke(111, upcallFuncAddr);
-			fail("Failed to throw out IllegalThreadStateException in the case of the invalid upcall during the trivial downcall.");
+			fail("Failed to throw out IllegalThreadStateException in the case of the invalid upcall during the Critical downcall.");
```

Comment:
Removing "out" would improve the message, and I don't think we need to capitalize "Critical".

Reply:
Fixed.